community_images/jmx-exporter/ironbank/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  jmx-exporter:
+    image: ${JMX_EXPORTER_IMAGE_REPOSITORY}:${JMX_EXPORTER_IMAGE_TAG}
+    ports:
+      - "12345:12345"
```

Comment:
Can we map host ports dynamically? `::12345`

Reply:
Done

Reply:
Done